frontend/src/components/Auth/firebase.js

```diff
@@ -15,6 +14,6 @@ const firebaseConfig = {
 
 // Initialize Firebase
 const app = initializeApp(firebaseConfig);
-export const auth = getAuth();
+export const auth = getAuth(app);
```

Comment:
got this issue actually this firebaseConfig doesn't have all the config please add this entry (databaseURL:import.meta.env.VITE_DATABASE_URL)